services/src/test/java/org/apache/druid/server/AsyncQueryForwardingServletTest.java

```diff
@@ -127,7 +127,11 @@ public class AsyncQueryForwardingServletTest extends BaseJettyTest
 {
   private static int port1;
   private static int port2;
-
+  private String errorMessage;
```

Comment:
The line above was also good to have a visual separation of the static and non-static fields.
```suggestion

  private String errorMessage;
```